### PR TITLE
Restyle selinux to fit other pages

### PR DESCRIPTION
**Pull request by marusak**
No difference functionally, just looks.

Before:
beforeselinux

Now:
newandhsiny

### CODE REVIEW

File: pkg/selinux/setroubleshoot-view.jsx [hunk 1]
```diff
@@ -462,29 +470,37 @@ export class SETroubleshootPage extends React.Component {
...
 
-        var errorMessage;
+        let errorMessage;
         if (this.props.error) {
             errorMessage = (
-                <Alert isInline
-                    variant='danger' title={this.props.error}
-                    actionClose={<AlertActionCloseButton onClose={this.handleDismissError} />} />
+                <AlertGroup isToast>
+                    <Alert
```
Comment (KKoukiou) [inline pkg/selinux/setroubleshoot-view.jsx:477]: When using Alerts inside Toast you should use isLiveRegion property which adds the aria-live="polite"  This helps assistive technoligies somehow.

**Review by KKoukiou — CHANGES_REQUESTED** [review]
only one tiny issue

File: pkg/selinux/setroubleshoot-view.jsx (hunk 1 above)
Comment (KKoukiou) [reply pkg/selinux/setroubleshoot-view.jsx:477]: Btw opened just now https://github.com/patternfly/patternfly-react/issues/5477

Comment (marusak) [reply pkg/selinux/setroubleshoot-view.jsx:477]: thanks, fixed!

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**